apps/rush-lib/src/logic/pnpm/PnpmShrinkwrapFile.ts

```diff
@@ -146,18 +147,25 @@ export class PnpmShrinkwrapFile extends BaseShrinkwrapFile {
   }
 
   /**
-   * Serializes the PNPM Shrinkwrap file
+   * Gets the version number from the list of top-level dependencies in the "dependencies" section
+   * of the shrinkwrap file
    */
-  protected serialize(): string {
```

Comment:
Can you move these back to where they were in this file to minimize the code change?

Reply:
Sure

Reply:
I want `getTopLevelDependencyVersion` exposed, so I had to move it above `protected` methods.